src/messages.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="messages.css">
+    <title>Messages</title>
```

Comment:
Keep the convention of always putting the css link tags right before end of `</head>`.
```suggestion
    <title>Messages</title>
    <link rel="stylesheet" href="messages.css">
```